.github/workflows/dotnet.yml

```diff
@@ -9,13 +9,14 @@ on:
 jobs:
   build:
     runs-on: windows-2019
+    timeout-minutes: 0
```

Comment:
This should not appear. We need to fix it.

Reply:
Comment Resolved.